cajita/unit_test/tstDynamicPartitioner.hpp

```diff
@@ -464,11 +475,11 @@ TEST( sparse_dim_partitioner, sparse_dim_partitioner_uniform_test )
 }
 TEST( sparse_dim_partitioner, sparse_dim_partitioner_random_tile_test )
```

Comment:
Since this file only tests the sparse version, let's call it `tstSparseDynamicPartitioner.hpp`. And then also create `tstDynamicPartitioner.hpp` for the dense case